### PR TITLE
Proper parenthetics

### PR DESCRIPTION
**Pull request by famavott**

### CODE REVIEW

File: src/parenthetics.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+"""Test if string of parentheses is open, balanced, or broken with the use of a data structure."""
+
+from stack import Stack
+
+
+def proper_parenthetics(string):
+    """Function tests if string is open, balanced, or broken."""
+    new_stack = Stack()
```
Comment (nhuntwalker) [inline src/parenthetics.py:8]: So why did you need the Stack again?

File: src/parenthetics.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        if char == '(':
+            count += 1
+        elif char == ')':
+            count -= 1
+        if count < 0:
+            return -1
+    if count == 0:
+        return 0
+    else:
+        return 1
```
Comment (nhuntwalker) [inline src/parenthetics.py:20]: This could be a ternary statement, like so:
```python
return 0 if count == 0 else 1
```

File: src/parenthetics.py (hunk 1 above)
Comment (famavott) [reply src/parenthetics.py:8]: Probably corpse code. I was trying to use a data structure, and that obviously didn't work out...
